### PR TITLE
Update log4j configuration for performance

### PR DESCRIPTION
**Pull request by jiacheliu3**
### What changes are proposed in this pull request?

Change the log format to improve performance. This impacts job master/worker log and audit log.

### Why are the changes needed?
`%F` is the file that log event is from (java file)
`%M` is the method
Example log looks like this
```
2021-12-14 17:26:32,763 INFO master.AbstractMaster (AbstractMaster.java:stop) - JobMaster: Stopping primary master.
2021-12-14 17:26:32,764 INFO master.AbstractMaster (AbstractMaster.java:stop) - JobMaster: Stopped primary master.
```

As log4j document specifies, using `%F` and `%M` will hurt performance. We have `%c{1}` which outputs the class name anyways so these info are not necessary.
https://logging.apache.org/log4j/2.x/manual/layouts.html#PatternFile in sections for `%F` and `%M`

### Does this PR introduce any user facing changes?

The log will look different. I don't think anyone is parsing those anyways so the impact should be small enough. We are just changing job master/worker log format to be the same as master/worker.

### CODE REVIEW

File: core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java [hunk 1]
```diff
@@ -620,7 +620,8 @@ public long getUsedBytes() {
         throw new InvalidArgumentException("Unrecognized worker range: " + workerRange);
     }
 
-    List<WorkerInfo> workerInfoList = new ArrayList<>();
+    List<WorkerInfo> workerInfoList = new ArrayList<>(
+        selectedLiveWorkers.size() + selectedLostWorkers.size());
```
Comment (jenoudet) [inline core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java:624]: Isn't this a duplicate of #14728 ?

Comment (jiacheliu3) [reply core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java:624]: Oops my branch messed up and I didn't notice. Will clean this up.

Comment (jiacheliu3) [reply core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java]: Removed that commit from my branch.

**Review by jenoudet — APPROVED** [review]

**Review by maobaolong — APPROVED** [review]
Thanks for this performance improvement, look forward to merge this, will use it in our cluster.

**Review by yuzhu — APPROVED** [review]

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Pull request merged**